authors/tests/base_test.py

```diff
@@ -430,3 +435,17 @@ def authentication_token_3(self, ):
         res = self.get_token_signup_user_3()
         token = res.data['token']
         return token
+
+    def edit_comment(self):
+        slug = self.create_article().data['slug']
+        url = reverse("articles:comment", kwargs={"slug": slug})
+        response = self.client.post(url, self.comment_data, format="json")
+        id = response.data['id']
```

Comment:
`id` is a reserved keyword in python, please change this variable name to `comment_id`.